### PR TITLE
DEVX-1940: Multi Region Clusters: augment tutorial in docs

### PR DESCRIPTION
**Pull request by ybyzek**
To view staged docs, please see link in https://github.com/confluentinc/docs/pull/5461

### CODE REVIEW

**Review by londoncalling — APPROVED** [review]
LGTM

**Review by addisonhuddy — APPROVED** [review]
Thanks for the improvements @ybyzek. This is great. 👍

File: multiregion/docs/multiregion.rst [hunk 1]
```diff
@@ -7,17 +7,59 @@ Overview
...
+You can choose how to replicate data, synchronously or asynchronously, on a per |ak| topic basis.
+It provides good durability guarantees and makes disaster recovery (DR) much easier.
+
+Benefits:
+
+- Supports hybrid deployments of synchronous and asynchronous replication between datacenters
+- Consumers can leverage data locality for reading |ak| data, which means better performance and lower cost
+- Ordering of |ak| messages is preserved across datacenters
+- Consumer offsets are preserved
+- In event of a disaster in a datacenter, new leaders are automatically elected in the other datacenter, and applications proceed without interruption. This achieves RTO=0 and RPO=0
```
Comment (jsancio) [inline multiregion/docs/multiregion.rst]: This is accurate if sync replication is used. If async replication is used then the user needs to perform an unclean leader election if all of the sync replicas are down.

**Review by jsancio — COMMENTED** [review]
Thanks for the improvements

File: multiregion/docs/multiregion.rst (hunk 1 above)
Comment (ybyzek) [reply multiregion/docs/multiregion.rst]: @jsancio good point, I'm augmenting the text to clarify.

**Pull request merged**